### PR TITLE
Fix: blurry basemaps in scene samples

### PR DESCRIPTION
**Pull request by yo1995**
This PR fixes the blurred basemaps in various samples that use the Brest, France coastline area, which became a navy base and pixelated/censored from the imagery basemap. issue at `common-samples/issues/2253`

| Affected sample | Action |
| -- | -- |
| Distance measurement analysis| update to use topographic map |
| Scene layer selection | update to use topographic map |
| Viewshed camera| update to Girona, Spain integrated mesh layer |
| Scene layer (URL)| no action needed, uses a topographic basemap |

image

### CODE REVIEW

**Review by vquach2404 — APPROVED** [review]
LGTM!

**Comment by saratchandrakarumuri** [comment]
@yo1995 Not directly related to change addressed in this PR. But since we are here can you update the brest scene layer used in `Distance measurement analysis` to the one used in `Scene layer (URL)`. Layer used distance sample has lot of bleeding and showing basemap which doesn't look good 

IMG_78B664648D63-1

**Review by philium — APPROVED** [review]

**Comment by yo1995** [comment]
> ...can you update the brest scene layer used in `Distance measurement analysis` to the one used in `Scene layer (URL)`. Layer used distance sample has lot of bleeding and showing basemap which doesn't look good

https://github.com/Esri/arcgis-runtime-samples-ios/blob/6d5ed31c11f03f6962b2c7527e117871fa75f190/arcgis-ios-sdk-samples/Analysis/Distance%20measurement%20analysis/DistanceMeasurementAnalysisViewController.swift#L38-L47

@philium In the `Distance measurement analysis` sample there are 2 elevation sources added. Do you remember why we need both of them, and if we need to remove one, which one should I remove? Thanks.

I tried to remove the Brest elevation source - the flakiness gets better, but not totally resolved. My guess is that the accuracy of elevation caused this issue.

**Comment by philium** [comment]
@yo1995 I do not know/remember why we used two elevation source. Presumably because it was specified in the design (if there is one) or because the reference implementation I consulted used two.

File: arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift
Comment (yo1995) [inline arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift:37]: According to Tanner, we substitute the elevation source with this service. I cannot really tell if the surface is elevated 🤔  - the Brest area is pretty flat as for this elevation source. But it does fixed the flaky-looking buildings in the scene.

**Comment by saratchandrakarumuri** [comment]
With this updated code, elevation is only applied to scene layer but not the scene itself and layer is rendered floating mid air

IMG_76C47025B2AF-1

**Comment by yo1995** [comment]
> With this updated code, elevation is only applied to scene layer but not the scene itself and layer is rendered floating mid air

Confirmed. So I really wonder if it is possible to correctly elevate these buildings. I'll wait for more feedback on this topic in the channel!

File: arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift
Comment (yo1995) [inline arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/DistanceMeasurementAnalysisViewController.swift:50]: Last week most people agreed to add an offset to the buildings layer to mitigate the clipping between elevated surface and buildings. Once the design get finalized, I'll request another review from you all.

**Comment by yo1995** [comment]
An 2m elevation offset has been added to the `DistanceMeasurementAnalysisViewController` so the scene layer doesn't clip into the terrain. Please feel free to take another look!

**Review by saratchandrakarumuri — APPROVED** [review]
This change looks good

**Review by vquach2404 — APPROVED** [review]
Changes look good!

**Pull request merged**
